web/src/index.css

```diff
@@ -102,7 +102,20 @@
 
 input[type='datetime-local']::-webkit-datetime-edit-text { color: darkcyan; }
 
+@media (max-width: 576px) {
+  .hidden-mobile {
+    display: none;
+  }
+}
+@media (min-width: 576px) {
+  .visible-mobile {
+    display: none;
+  }
+}
 
+.hidden {
```

Comment:
Can we use Tailwind's hidden?
https://tailwindcss.com/docs/display#hidden

Reply:
Done! 👍🏽